doc/nrf/libraries/security/nrf_security/doc/configuration.rst

```diff
@@ -19,6 +19,8 @@ PSA crypto support is included by default when you enable nRF Security through t
 PSA crypto support is provided through PSA Crypto APIs and is implemented by PSA core.
 PSA core uses PSA drivers to implement the cryptographic features either in software, or using hardware accelerators.
 
+Version 1.0 of the `PSA Crypto API <https://arm-software.github.io/psa-api/>`_ is implemented when building with TF-M, otherwise version 1.2 is implemented.
```

Comment:
```suggestion
When you :ref:`build with TF-M<ug_tfm>`, `PSA Certified Crypto API`_ v1.0 is implemented.
Otherwise, when you build without TF-M, v1.2 of the API is used.
```
Though I guess this only applies when you build nRF Security with TF-M? If so, this line is more correct:
```suggestion
When you build your application with nRF Security and :ref:`TF-M included<ug_tfm>`, `PSA Certified Crypto API`_ v1.0 is implemented.
Otherwise, when you build without TF-M, v1.2 of the API is used.
```

Reply:
It applies to any NCS user, so I applied the first commit.

Thank you for the review.